### PR TITLE
feat: Adding Footer to the layout

### PR DESCRIPTION
**Pull request by titan-rocky**
Changes :
- Added a footer for the website
- replaced the dummy footer div in the main page component with the created footer
- added "Glacial Indifference" font for the bitspace logo in tailwind.config and globals.css

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import Link from "next/link"
+import { useState } from "react"
```
Comment (MKMukeshkannan) [inline src/components/Footer.tsx]: - remove this un-used import

**Review by MKMukeshkannan — CHANGES_REQUESTED** [review]
Follow the commit message convention.

**Comment by MKMukeshkannan** [comment]
Good Work 👍

**Comment by titan-rocky** [comment]
Thank you ❤️

**Pull request merged**